src/gui/sdlmain.cpp

```diff
@@ -3420,77 +3637,107 @@ static void GUI_StartUp(Section *sec)
+			                  "capmouse",
+			                  "Cap Mouse");
 
 		// Apply the user's mouse sensitivity settings
 		Prop_multival *p3 = section->Get_multival("sensitivity");
-		sdl.mouse.xsensitivity = p3->GetSection()->Get_int("xsens");
-		sdl.mouse.ysensitivity = p3->GetSection()->Get_int("ysens");
+		MOUSE_SetSensitivity(p3->GetSection()->Get_int("xsens"),
+		                     p3->GetSection()->Get_int("ysens"));
 
 		// Apply raw mouse input setting
 		SDL_SetHintWithPriority(SDL_HINT_MOUSE_RELATIVE_MODE_WARP,
 		                        section->Get_bool("raw_mouse_input") ? "0" : "1",
 		                        SDL_HINT_OVERRIDE);
 	}
-	LOG_MSG("SDL: Mouse %s%s.", mouse_control_msg.c_str(), middle_control_msg.c_str());
+	LOG_MSG("SDL: Mouse %s%s.",
+	        mouse_control_msg.c_str(),
+	        middle_control_msg.c_str());
 
 	/* Get some Event handlers */
-	MAPPER_AddHandler(GFX_RequestExit, SDL_SCANCODE_F9, PRIMARY_MOD,
-	                  "shutdown", "Shutdown");
-	MAPPER_AddHandler(SwitchFullScreen, SDL_SCANCODE_RETURN, MMOD2,
-	                  "fullscr", "Fullscreen");
-	MAPPER_AddHandler(Restart, SDL_SCANCODE_HOME, MMOD1 | MMOD2, "restart",
-	                  "Restart");
+	MAPPER_AddHandler(GFX_RequestExit, SDL_SCANCODE_F9, PRIMARY_MOD, "shutdown", "Shutdown");
+	MAPPER_AddHandler(SwitchFullScreen, SDL_SCANCODE_RETURN, MMOD2, "fullscr", "Fullscreen");
+	MAPPER_AddHandler(Restart, SDL_SCANCODE_HOME, MMOD1 | MMOD2, "restart", "Restart");
 #if C_DEBUG
 /* Pause binds with activate-debugger */
 #else
-	MAPPER_AddHandler(&PauseDOSBox, SDL_SCANCODE_PAUSE, MMOD2,
-	                  "pause", "Pause Emu.");
+	MAPPER_AddHandler(&PauseDOSBox, SDL_SCANCODE_PAUSE, MMOD2, "pause", "Pause Emu.");
 #endif
 	/* Get Keyboard state of numlock and capslock */
 	SDL_Keymod keystate = SDL_GetModState();
 
 	// A long-standing SDL1 and SDL2 bug prevents it from detecting the
 	// numlock and capslock states on startup. Instead, these states must
 	// be toggled by the user /after/ starting DOSBox.
-	startup_state_numlock = keystate & KMOD_NUM;
+	startup_state_numlock  = keystate & KMOD_NUM;
 	startup_state_capslock = keystate & KMOD_CAPS;
 }
 
-static void HandleMouseMotion(SDL_MouseMotionEvent * motion) {
-	if (mouse_is_captured || sdl.mouse.control_choice == Seamless)
-		Mouse_CursorMoved((float)motion->xrel*sdl.mouse.xsensitivity/100.0f,
-						  (float)motion->yrel*sdl.mouse.ysensitivity/100.0f,
-						  (float)(motion->x-sdl.clip.x)/(sdl.clip.w-1)*sdl.mouse.xsensitivity/100.0f,
-						  (float)(motion->y-sdl.clip.y)/(sdl.clip.h-1)*sdl.mouse.ysensitivity/100.0f,
-						  mouse_is_captured);
+static void HandleMouseMotion(SDL_MouseMotionEvent *motion)
+{
+	if (motion->xrel < INT16_MIN || motion->xrel > INT16_MAX ||
+	    motion->yrel < INT16_MIN || motion->yrel > INT16_MAX || motion->x < 0 ||
+	    motion->x > UINT16_MAX || motion->y < 0 || motion->y > UINT16_MAX) {
+		LOG_ERR("SDL: Unexpected motion values: rel %dx%d, abs %dx%d",
```

Comment:
minor: `x` as a separator is a bit weird as it implies dimensions, no?

Reply:
Changed to coma.